js-exercises/07-js-loops/index.js

```diff
@@ -0,0 +1,32 @@
+"use strict"
+
+let sum = 0;
+for(let i = 0; i <= 1000; i++){
+    sum += i;
+    if( sum > 400){
+        console.log("Partial result: " + sum);
+        break;
+    }
+}
+
+for(let i = 0; i <= 10; i++){
+    console.log(i);
+}
+
+let i = 100;
+do{
+    console.log(i);
+    i--;
+}while(i>0);
+
+let x = 1;
+while(x < 100){
+    console.log(x);
+    x++;
+}
+
+let star = '';
+for(let i = 0; i < 13; i++){
```

Comment:
Do you get the desired output from this code?

Reply:
I think so, please see the image below:
![image](https://user-images.githubusercontent.com/1565878/95776144-620cf000-0c89-11eb-8968-6dee7f3be64d.png)